tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/LsMetricsMonitor.java

```diff
@@ -74,7 +74,7 @@ private LsMetricsMonitor(final String metrics, final DataStore store) {
                 final long newstrt = System.nanoTime();
                 stats.addValue(
                     (double) (newcount - count) /
-                        (double) TimeUnit.SECONDS.convert(newstrt - start, TimeUnit.NANOSECONDS)
+                        (double) TimeUnit.SECONDS.convert(Math.max(newstrt - start, 1_000_000_000), TimeUnit.NANOSECONDS)
```

Comment:
since we are targeting a `double`, would it make sense to handle the math ourselves instead of relying on `TimeUnit#convert` that is limited to `long`? This would allow us to handle sub-second captures more accurately too.

```suggestion
                        (((double) Math.max(newstrt - start, 1)) / 1_000_000_000)
```
~~~

Reply:
I understand your point but I think that using TimeUnit.convert express better that we are transforming something expressed in nanosecond to seconds, in the second case the intention it's more subtle

Reply:
Is my thought an overkill? WDYT?

Reply:
@yaauie LMKWDT about the to put the explicit intention using the type